app/src/main/java/ch/epfl/sweng/SDP/game/WaitingPageActivity.java

```diff
@@ -207,22 +213,25 @@ protected void onCreate(Bundle savedInstanceState) {
                 WordNumber.TWO);
 
         Typeface typeMuro = Typeface.createFromAsset(getAssets(), "fonts/Muro.otf");
-        ((TextView) findViewById(R.id.playersReadyText)).setTypeface(typeMuro);
-        ((TextView) findViewById(R.id.playersCounterText)).setTypeface(typeMuro);
-        ((TextView) findViewById(R.id.buttonWord1)).setTypeface(typeMuro);
-        ((TextView) findViewById(R.id.buttonWord2)).setTypeface(typeMuro);
-        ((TextView) findViewById(R.id.voteText)).setTypeface(typeMuro);
-        ((TextView) findViewById(R.id.roomID)).setText(format("Room ID: %s", roomID));
+
+        setTypeFace(typeMuro, findViewById(R.id.playersReadyText),
+                            findViewById(R.id.playersCounterText), findViewById(R.id.buttonWord1),
+                            findViewById(R.id.buttonWord2), findViewById(R.id.voteText),
+                            findViewById(R.id.waitingTime));
+
+        findViewById(R.id.waitingTime).setVisibility(View.GONE);
 
     }
 
-    //TODO Give this path
-//        // Select the words based on the user current league
-//        DatabaseReference wordsSelectionRef = database
-//                .getReference(format(Locale.getDefault(), "leagues.%s.%s",
-//                        Account.getInstance(this).getCurrentLeague(),
-//                        WORD_CHILDREN_DB_ID));
-//        wordsSelectionRef.addListenerForSingleValueEvent(listenerWords);
+    private void setTypeFace(Typeface typeface, View ...views) {
+        for(View view: views) {
+            ((TextView) view).setTypeface(typeface);
+        }
+    }
```

Comment:
This is a good idea ! Maybe you can move it to Activity.java because there are similar methods there (eg: public void setVisibility(int visibility, int... ids))